### DISCUSSION TITLE
Pushes to propose modifications to the user, around

### DISCUSSION THREAD

**Discussion by skr3ch** [streetcomplete/StreetComplete#5680] [Ideas]
Hello, 
First of all, sorry if already exists this discussion.
My concern is that I really like the app and the idea behind. But everybody "have a life". While someone is outside house, do quests is the last though in mind.
What if sometimes the app locates position by GPS, and proposes quests by push?
I understand the difficulty for developing. But is an idea. Without a feature like this, maybe I uninstall the app because I'm not using it to fill data.

**Comment by FloEdelmann** [comment]
* See also https://github.com/streetcomplete/StreetComplete/issues/48

**Reply by mnalis** [reply]
Just to note that [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (StreetComplete _"Expert Edition"_ fork) does implement that feature, so @skr3ch (and other interested people) might want to try it out (SCEE can be installed alongside StreetComplete, and by default looks very similar unless you enable extra features in the settings).

I did try it at the time, but found it "too chatty". But then, I'm in urban areas where there are _always_ some quests to be solved. 

In other words, unless the (probably very rural) areas one is in are quite clean of StreetComplete tasks (and/or one creates preset with most quests disabled), one might find such feature more annoying than helpful (if you get a _"There is a new quest"_ notification every 30 seconds, it gets boring _very_ fast). 

Also note that (by design) since app must be active in background at all times in such case (instead of just when it is in foreground with screen on when the feature is disabled), it will drain phone battery more quickly.
